src/Cardano/Prelude/Orphans.hs

```diff
@@ -1,35 +1,25 @@
+{-# LANGUAGE CPP               #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE PolyKinds         #-}
 {-# LANGUAGE TypeFamilies      #-}
 
 {-# OPTIONS_GHC -fno-warn-orphans #-}
+{-# OPTIONS_GHC -fno-warn-unused-imports #-}
```

Comment:
What imports are being warned about?

Reply:
Oops.  That was a carry over from a previous commit.  I'll remove that.

Reply:
Fixed!